doc/CONTRIBUTING.md

```diff
@@ -2,11 +2,11 @@
 
 This project uses a `luigi` pipeline to generate the QGreenland package. Luigi
 is a workflow management system developed by Spotify:
-https://github.com/spotify/luigi
+[https://github.com/spotify/luigi](https://github.com/spotify/luigi).
 
 This project is currently in early development stages, so expect rapid change.
-Nothing is written in stone! Releases can be found at
-https://qgreenland.org/explore!
+Nothing is written in stone! Stable releases can be found at
+[https://qgreenland.org/explore](https://qgreenland.org/explore)!
```

Comment:
We'll have to update this link when the the new version of the website goes live.